wagn/rails/controllers/card_controller.rb

```diff
@@ -248,7 +243,9 @@ def show view = nil, status = 200
     else
       args = { :text=>result, :status=>status }
       args[:content_type] = 'text/text' if format == :file
-      render args
+      card.run_callbacks :render_view do
```

Comment:
why the name "render_view"?  seems a little misleading, since it doesn't get called whenever we render a view.  should it just be "read" or "show" maybe?